### PR TITLE
download txt file_for datatree

### PR DESCRIPTION
**Pull request by yoouung**
- add download function for detected, converted result downloading
- fix bug where 유형 is printed as all detected errors in the `detected`, `converted` table if more than one error is detected in a single sentence

### CODE REVIEW

**Comment by yoouung** [comment]
### bug fix
image
-> 
image

**Comment by yoouung** [comment]
### converted txt file
image

File: dt-demo-fe/src/components/result/downloadBtn.js [hunk 1]
```diff
@@ -23,8 +23,16 @@ const DownloadBtn = ({ type, pdfRef }) => {
     };
 
     // TODO: implement me
```
Comment (doorooful) [inline dt-demo-fe/src/components/result/downloadBtn.js:25]: Can you remove this line?

File: dt-demo-fe/src/pages/converter/converted.js [hunk 2]
```diff
@@ -115,12 +116,33 @@ const ConverterConversionResult = () => {
...
+                ];
+
+                for (let j = 0; j < expected_matches.length; j++) {
+                    const id = matches[j][2];
+                    const expected = expected_matches[j][2];
+
+                    let conv_final = converted_highlighted_sentences;
+                    if (converted_highlighted_sentences.includes("<del>")) {
+                        // TODO: check style
+                        // cstyle += ";text-decoration: line-through;";
```
Comment (doorooful) [inline dt-demo-fe/src/pages/converter/converted.js:140]: Can we remove this line unless this line would be used?

File: dt-demo-fe/src/pages/converter/converted.js [hunk 3]
```diff
@@ -139,12 +161,13 @@ const ConverterConversionResult = () => {
                         }
                     }
 
-                    let conv_final = converted_highlighted_sentences;
-
                     if (converted_highlighted_sentences.includes("<del>")) {
                         // TODO: check style
                         cstyle += ",text-decoration: line-through;";
-                        conv_final = converted_highlighted_sentences.replace(/<del>/g, "");
+                        conv_final = converted_highlighted_sentences.replace(
```
Comment (doorooful) [inline dt-demo-fe/src/pages/converter/converted.js:167]: I don't think there's any changes here.

File: dt-demo-fe/src/pages/converter/converted.js [hunk 4]
```diff
@@ -179,9 +202,9 @@ const ConverterConversionResult = () => {
                             />
                         </tr>
                     );
+                    i += 1;
```
Comment (doorooful) [inline dt-demo-fe/src/pages/converter/converted.js:205]: Can you add details on `let i = 1;` about the usage?
(about what this variable tracks for)

File: dt-demo-fe/src/pages/converter/detected.js [hunk 5]
```diff
@@ -213,7 +225,7 @@ const ConverterDetectionResult = () => {
             lineHeight: "1.5",
         },
         th: {
-            padding: "10px",
+            // modify me
```
Comment (doorooful) [inline dt-demo-fe/src/pages/converter/detected.js:228]: May be we could add `TODO: ` here to find easily on `ctrl + shift + f`

**Review by doorooful — APPROVED** [review]
Please find my comments ◡̈ Thanks!

**Pull request merged**